draft-birkholz-scitt-architecture.md

```diff
@@ -162,56 +162,56 @@ Claim:
 
-Feed:
+Ledger:
 
-: An identifier chosen by the Issuer for the Artifact. For every Issuer and Feed, the Ledger on a Transparency Service contains a sequence of Claims about the same Artifact.
- In COSE, Feed is one header attributes in the protected header of the Envelope.
+: an optional verifiable data structure that stores Claims in a Receipt Service. SCITT supports multiple Ledger formats to accommodate different Receipt Service implementations, such as historical Merkle Trees and sparse Merkle Trees. Not all Receipt Services require a Ledger.
 
-Ledger:
+Feed:
 
-: the verifiable data structure that stores Claims in a transparency service. SCITT supports multiple Ledger formats to accommodate different transparency service implementations, such as historical Merkle Trees and sparse Merkle Trees.
+: An identifier chosen by the Issuer for the Artifact. For every Issuer and Feed, the Ledger on a Receipt Service contains a sequence of Claims about the same Artifact.
+In COSE, Feed is one header attributes in the protected header of the Envelope.
 
-Transparency Service:
+Receipt Service (RS):
 
-: the entity that maintains and extends the Ledger, and endorses its state. A Transparency Service can be a complex distributed system, and SCITT requires the TS to provide many security guarantees about its Ledger. The identity of a TS is captured by a public key that must be known by Verifiers in order to validate Receipts.
+: the entity that maintains and extends the Ledger, and endorses its state. A Receipt Service can range from something as simple as a timestamping service to a more complex distributed system. The security properties of the RS depend on the type of verifiability it affords to verifiers. The identity of a RS is captured by a public key that must be known by Verifiers in order to validate Receipts.
 
 Receipt:
 
-: a Receipt is a special form of COSE countersignature for Claims that embeds cryptographic evidence that the Claim is recorded in the Ledger. It consists of a Ledger-specific inclusion proof, a signature by the Transparency Service of the state of the Ledger, and additional metadata (contained in the countersignature protected headers) to assist in auditing.
+: a Receipt is a special form of COSE countersignature for Claims that embeds cryptographic evidence that the Claim is recorded in the Ledger. It consists of a Ledger-specific inclusion proof, a signature by the Receipt Service of the state of the Ledger, and additional metadata (contained in the countersignature protected headers) to assist in auditing.
 
 Registration:
 
-: the process of submitting a Claim to a Transparency Service, applying its registration policy, storing it in the Ledger and producing the Receipt returned to the submitter.
+: the process of submitting a Claim to a Receipt Service, applying its registration policy, storing it in the Ledger and producing the Receipt returned to the submitter.
```

Comment:
(since the ledger is optional, appending the claim to the ledger is also optional)